proto/stream_plan.proto

```diff
@@ -52,6 +52,7 @@ message MaterializeNode {
   repeated plan.ColumnOrder column_orders = 3;
   // Column IDs of input schema
   repeated int32 column_ids = 4;
+  // TODO: remove this field, not used anywhere
   repeated int32 distribution_keys = 5;
```

Comment:
I think this can be removed because the distribution key of materialized view must be PKs (`column_orders`) now. cc. @xx01cyx

Reply:
exactly

Reply:
skyzh: Next time for sure!